<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="central">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="switcher" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="actions">
         <property name="text">
          <string>...</string>
         </property>
         <property name="popupMode">
          <enum>QToolButton::MenuButtonPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="files_switcher">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stacked">
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionOpen"/>
    <addaction name="actionRecent_files"/>
    <addaction name="actionRecent_projects"/>
    <addaction name="separator"/>
    <addaction name="actionClose_all_projects_or_editer"/>
    <addaction name="separator"/>
    <addaction name="actionSave_file"/>
    <addaction name="actionSave_current_file_as"/>
    <addaction name="actionSave_all_files"/>
    <addaction name="separator"/>
    <addaction name="actionClose_current_file"/>
    <addaction name="actionClose_all_files"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit_E">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionUodo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionRemove"/>
    <addaction name="actionDelete"/>
    <addaction name="actionPaste_from_hist_Clippad"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_all"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuBuild_B">
    <property name="title">
     <string>Build(&amp;B)</string>
    </property>
    <addaction name="actionBuild_all_project"/>
    <addaction name="actionBuild_current_project"/>
    <addaction name="actionBuild_current_file"/>
    <addaction name="separator"/>
    <addaction name="actionRebuild_all_projects"/>
    <addaction name="actionRebuild_current_project"/>
    <addaction name="actionRebuild_current_file"/>
    <addaction name="actionClear_all_project"/>
    <addaction name="actionClear_current_project"/>
    <addaction name="actionClear_current_file"/>
    <addaction name="separator"/>
    <addaction name="actionCancel_the_build"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_build_manager"/>
   </widget>
   <widget class="QMenu" name="menuDebug_D">
    <property name="title">
     <string>Debug(&amp;D)</string>
    </property>
    <addaction name="actionStart_debug"/>
    <addaction name="actionInterrupt"/>
    <addaction name="actionContinue"/>
    <addaction name="actionStop_debugging"/>
    <addaction name="actionRestart_debuggin"/>
    <addaction name="separator"/>
    <addaction name="actionGo_next"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_break_point"/>
    <addaction name="actionRemove_break_point"/>
    <addaction name="actionRemove_all_break_points"/>
   </widget>
   <widget class="QMenu" name="menuTool_T">
    <property name="title">
     <string>Tool(&amp;T)</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout_protocol_designer"/>
    <addaction name="actionReport_bugs"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_update"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuEdit_E"/>
   <addaction name="menuBuild_B"/>
   <addaction name="menuDebug_D"/>
   <addaction name="menuTool_T"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="project_dock">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="project_tree">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="files_dock">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeWidget_2">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="element_dock">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="elements">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="property_dock">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ouput_dock">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="console">
        <attribute name="title">
         <string>Console</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="output"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="find">
        <attribute name="title">
         <string>Find</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>634</width>
              <height>191</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionCreate">
   <property name="text">
    <string>Create new protocol describle file or project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open new protocol describle file or project</string>
   </property>
  </action>
  <action name="actionRecent_files">
   <property name="text">
    <string>Recent files</string>
   </property>
  </action>
  <action name="actionRecent_projects">
   <property name="text">
    <string>Recent projects</string>
   </property>
  </action>
  <action name="actionClose_all_projects_or_editer">
   <property name="text">
    <string>Close all projects or editer</string>
   </property>
  </action>
  <action name="actionSave_file">
   <property name="text">
    <string>Save file</string>
   </property>
  </action>
  <action name="actionSave_current_file_as">
   <property name="text">
    <string>Save current file as ...</string>
   </property>
  </action>
  <action name="actionSave_all_files">
   <property name="text">
    <string>Save all files</string>
   </property>
  </action>
  <action name="actionClose_current_file">
   <property name="text">
    <string>Close current file</string>
   </property>
  </action>
  <action name="actionClose_all_files">
   <property name="text">
    <string>Close all files</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUodo">
   <property name="text">
    <string>Uodo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionPaste_from_hist_Clippad">
   <property name="text">
    <string>Paste from Clipboard history</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionBuild_all_project">
   <property name="text">
    <string>Build all projects</string>
   </property>
  </action>
  <action name="actionBuild_current_project">
   <property name="text">
    <string>Build current project</string>
   </property>
  </action>
  <action name="actionBuild_current_file">
   <property name="text">
    <string>Build current file</string>
   </property>
  </action>
  <action name="actionRebuild_all_projects">
   <property name="text">
    <string>Rebuild all projects</string>
   </property>
  </action>
  <action name="actionRebuild_current_project">
   <property name="text">
    <string>Rebuild current project</string>
   </property>
  </action>
  <action name="actionRebuild_current_file">
   <property name="text">
    <string>Rebuild current file</string>
   </property>
  </action>
  <action name="actionClear_all_project">
   <property name="text">
    <string>Clear all project</string>
   </property>
  </action>
  <action name="actionClear_current_project">
   <property name="text">
    <string>Clear current project</string>
   </property>
  </action>
  <action name="actionClear_current_file">
   <property name="text">
    <string>Clear current file</string>
   </property>
  </action>
  <action name="actionCancel_the_build">
   <property name="text">
    <string>Cancel building</string>
   </property>
  </action>
  <action name="actionOpen_build_manager">
   <property name="text">
    <string>Open build manager</string>
   </property>
  </action>
  <action name="actionAbout_protocol_designer">
   <property name="text">
    <string>About protocol designer</string>
   </property>
  </action>
  <action name="actionReport_bugs">
   <property name="text">
    <string>Report bugs</string>
   </property>
  </action>
  <action name="actionCheck_for_update">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionStart_debug">
   <property name="text">
    <string>Start debugging</string>
   </property>
  </action>
  <action name="actionInterrupt">
   <property name="text">
    <string>Interrupt</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="text">
    <string>Continue</string>
   </property>
  </action>
  <action name="actionStop_debugging">
   <property name="text">
    <string>Stop debugging</string>
   </property>
  </action>
  <action name="actionRestart_debuggin">
   <property name="text">
    <string>Restart debuggin</string>
   </property>
  </action>
  <action name="actionGo_next">
   <property name="text">
    <string>Go next</string>
   </property>
  </action>
  <action name="actionAdd_break_point">
   <property name="text">
    <string>Add break point</string>
   </property>
  </action>
  <action name="actionRemove_break_point">
   <property name="text">
    <string>Remove break point</string>
   </property>
  </action>
  <action name="actionRemove_all_break_points">
   <property name="text">
    <string>Remove all break points</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionOpen_In_Explore">
   <property name="text">
    <string>Open In Explore</string>
   </property>
  </action>
  <action name="actionAdd_Existed">
   <property name="text">
    <string>Add Existed Files</string>
   </property>
  </action>
  <action name="actionRemove_File">
   <property name="text">
    <string>Remove File</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionClose_Current_Project">
   <property name="text">
    <string>Close Current Project</string>
   </property>
  </action>
  <action name="actionClose_All_Projects">
   <property name="text">
    <string>Close All Projects</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionCollapse_All">
   <property name="text">
    <string>Collapse All</string>
   </property>
  </action>
  <action name="actionCopy_Full_Path">
   <property name="text">
    <string>Copy Full Path</string>
   </property>
  </action>
  <action name="actionCopy_File_Name">
   <property name="text">
    <string>Copy File Name</string>
   </property>
  </action>
  <action name="actionSave_Current_File_As">
   <property name="text">
    <string>Save Current File As</string>
   </property>
  </action>
  <action name="actionSave_All_Files">
   <property name="text">
    <string>Save All Files</string>
   </property>
  </action>
  <action name="actionReload_File">
   <property name="text">
    <string>Reload File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
